packages/binding-http/src/http-server.ts

```diff
@@ -529,7 +539,27 @@ export default class HttpServer implements ProtocolServer {
                   thing.readProperty(segments[3], options)
                   // property.read(options)
                     .then((value:any) => {
-                      let content = ContentSerdes.get().valueToContent(value, <any>property);
+                      // try to find contentType (How to do this better)
+                      // Should readProperty() return an encapsulated value container with value&contenType
+                      // as sketched in https://github.com/w3c/wot-scripting-api/issues/201#issuecomment-573702999
+                      let contentType;
```

Comment:
That's a really good point, I'm going to follow w3c/wot-scripting-api#201 and try to contribute there. From the developer's point of view, the problem is how to know when to DataSchema make sense and when is not.

Reply:
Agree. This is a *broader* issue that needs to be solved in the Scripting API itself.